### DISCUSSION TITLE
Help!!! MetaMask Sweeping Restoration Needed

### DISCUSSION THREAD

**Discussion by eduviehenry** [flashbots/pm#104] [general]
To whom it may concern,

I realized I have used my MetaMask password on too many sites so I decided to change it to something much stronger. 
So, on the 28 Dec 2021, I re-imported my MetaMask wallet with the new password using a brave extension I added from the official google chrome extension website on the same day. Meanwhile, on the 10 Jan 2022 I checked my MM wallet address on  etherscan  and I realized all my ERC20 and BEP 20 tokens were swept away from my wallet on 01 Jan 2022 without my knowledge. I reported the incident to MetaMask immediately and they hinted I must have installed the MM wallet from a dodgy website hence they cannot help. 

Meanwhile, I have tokens (BAT, KNC, SFI, RGT, MIR, CAKE, 1INCH & BNB) I am currently staking on different pools and DeFi platforms worth roughly over £5000 based on current market price.

Is there anyone here that can help me recover my all my tokens at a 10% commission as I was refereed to this platform by MyCrypto.com?

I look forward to hearing from you all on this very urgent matter

Thanks

Regards

Henry Eduvie

**Comment by gidomason** [comment]
You need public your public key for details info at first.
